doc/tutorials/content/sources/executor/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 
 project(executor)
 
-find_package(PCL 1.11.1.99 COMPONENTS common filters visualization REQUIRED)
+find_package(PCL 1.11.1 COMPONENTS common filters visualization REQUIRED)
```

Comment:
Why are you setting this to 1.11.1? You're implicitly saying "it works from 1.11.1" but executors are not part of the 1.11.1 release.

Reply:
That'd be due to confusion caused by me